src/rpc/net.cpp

```diff
@@ -224,7 +224,10 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
         obj.pushKV("masternode", stats.m_masternode_connection);
         obj.pushKV("startingheight", stats.nStartingHeight);
         if (fStateStats) {
-            obj.pushKV("banscore", statestats.m_misbehavior_score);
+            if (IsDeprecatedRPCEnabled("banscore")) {
+                // banscore is deprecated in v0.21 for removal in v0.22
```

Comment:
Dash uses other version numberation for v21, v22

```suggestion
                // banscore is deprecated in v21 for removal in v22
```

Reply:
Implemented Suggestion

Reply:
you did this in the wrong commit; but I think it's fine enought